main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -1132,7 +1133,17 @@ static private void createProjectSynchronously(
                              .stream()
                              .limit(INFER_ROW_LIMIT)
                              .collect(Collectors.toList());
-                    rows.forEach(r->listCells.add(r.cells.toArray()));
+                    
+                    // convert the null object to prevent the NPE
+                    for (Row row : rows) {
+                        for (int i = 0; i < row.cells.size(); i++) {
+                            Cell cell = row.cells.get(i);
+                            if (cell == null) {
+                                row.cells.set(i, new Cell(StringUtils.EMPTY, null));
+                            }
+                        }
+                        listCells.add(row.cells.toArray());
+                    }
```

Comment:
I'm confused - wasn't this change already merged?

Reply:
This is part of result of the merge. It won't do anything after merge back to master.